### PR TITLE
Feature-HowTo-Extensions

### PR DESCRIPTION
**Pull request by sunilarjun**
Initial commit with structure and updating sidebar showing new how-to page for extensions.

### CODE REVIEW

**Comment by sunilarjun** [comment]
This PR addresses #187

File: docs/how-to-guides/installing-uninstalling-extensions.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
+---
+title: Installing and Uninstalling Rancher Desktop Extensions
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import TabsConstants from '@site/core/TabsConstants';
+
+Rancher Desktop introduces **Extensions** with the new `v1.9.0-tech-preview` release. The implementation allows for users to extend Rancher Desktop's functionality to better suit their needs. This guide will demonstrate how to install and uninstall extensions on the application.
```
Comment (gunamata) [inline docs/how-to-guides/installing-uninstalling-extensions.md:9]:
```suggestion
The **Extensions** feature introduced in the `v1.9.0-tech-preview` release enables you to use **Docker Desktop Extensions** within Rancher Desktop. The feature helps you extend Rancher Desktop's functionality to meet your additional needs. This guide will demonstrate how to install and uninstall Docker extensions in Rancher Desktop.
```

**Review by gunamata — CHANGES_REQUESTED** [review]
Just modified the intro part to make it clear that it's Docker extensions that users can install within Rancher Desktop.

File: docs/how-to-guides/installing-uninstalling-extensions.md (hunk 1 above)
Comment (sunilarjun) [reply docs/how-to-guides/installing-uninstalling-extensions.md]: Just updated the intro with new commit, thanks for the review Guna!

File: docs/how-to-guides/installing-uninstalling-extensions.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+1. Click **Extensions** from the main UI to navigate to the **Catalog** tab. Here you can search through extensions available for use.
+
+![](https://suse-rancher-media.s3.us-east-1.amazonaws.com/desktop/1.9-tech-preview/how-to/Windows_Extensions.png)
+
+2. When you find an extension you want to use, click on the extension to open the **Detail** view, then click on the **Install** button on the right hand side as seen in the image below. 
+
+![](https://suse-rancher-media.s3.us-east-1.amazonaws.com/desktop/1.9-tech-preview/how-to/Windows_Extensions-Details.png)
+
+3. Please shutdown and restart your Rancher Desktop application for changes to take effect.
```
Comment (gunamata) [inline docs/how-to-guides/installing-uninstalling-extensions.md:32]: We don't have to shutdown and restart Rancher Desktop to see the changes after installing an extension.

**Review by gunamata — APPROVED** [review]
LGTM

**Pull request merged**